circuits/CHANGELOG.md

```diff
@@ -3,6 +3,102 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.2.0](https://github.com/privacy-scaling-explorations/maci/compare/v1.1.1...v1.2.0) (2024-02-23)
```

Comment:
@ctrlc03 are these autogenerated changelogs, or did you add this manually?

Reply:
lerna auto generates when I did the bump. For the release we can curate a list of the most important ones